app/views/layouts/application.html.erb

```diff
@@ -52,6 +52,7 @@
 
     <%= yield(:head) %>
 
+    <script src="https://cdn.jsdelivr.net/npm/frappe-charts@0.0.8/dist/frappe-charts.min.iife.js"></script>
```

Comment:
Ah, could we include this /only/ on the profile pages? That way it's only when necessary. Make sense?

Reply:
okk @jywarren I would do that